lib/logstash/outputs/opensearch/http_client/pool.rb

```diff
@@ -433,15 +443,13 @@ def set_last_es_version(version, url)
     end
 
     def set_maximum_seen_major_version(major)
-      if major >= 6
-        @logger.warn("Detected a 6.x and above cluster: the `type` event field won't be used to determine the document _type", es_version: major)
-      end
+      @logger.warn("Detected a 6.x and above cluster: the `type` event field won't be used to determine the document _type", es_version: major)
```

Comment:
Why do we need this warning?

Reply:
Kept warning. Will scan for relevancy in next PR.